pkg/mover/chart.go

```diff
@@ -131,48 +128,59 @@ type ChartMoveRequest struct {
 	Target Target
 }
 
-// ChartMover represents a Helm Chart moving relocation. It's initialization must be done view NewChartMover
-type ChartMover struct {
-	chartDestination        string
-	imageChanges            []*internal.ImageChange
-	chartChanges            []*internal.RewriteAction
-	sourceContainerRegistry internal.ContainerRegistryInterface
-	targetContainerRegistry internal.ContainerRegistryInterface
-	targetOfflineTarPath    string
-	chart                   *chart.Chart
-	logger                  Logger
-	retries                 uint
+// ChartData is the part of the Chart mover data that should be saved for a move to complete
+// without any network access to the original chart resources
+type ChartData struct {
```

Comment:
Move this abstraction to intermediate.go

Reply:
Done